better-mouse-trap/README.md

```diff
@@ -2,6 +2,103 @@
 
 A smarter mouse trap with SMS alerting.
 
-![](../app-banners/nf3.png)
+Find the complete story in the detailed [Hackster.io writeup].
 
-You can find all instructions for building this project in [its Hackster writeup](https://www.hackster.io/hendersoncarlton/i-love-checking-on-mousetraps-said-no-one-ever-52c5e7). This repo contains the project’s [JSONata configuration](./mousetrap.jsonata) for reference.
\ No newline at end of file
+[![Mouse wearing a helmet grabbing cheese from a trap](../app-banners/nf3.png)][hackster.io writeup]
+
+## TL;DR Instructions
```

Comment:
Is this heading necessary?

I feel like you already imply that this is the quick-and-dirty version by linking to Hackster for the “complete” version earlier.